logstash-core/lib/logstash/config/source/local.rb

```diff
@@ -90,6 +94,13 @@ def path
         end
       end
 
+      def get_unmatched_files
+        # transform "/var/lib/*.conf" => /var/lib/*
+        t = File.split(@path)
+        all_files = Dir.glob(File.join(t.first, "*")).sort
```

Comment:
This will not show hidden (dot files) in the list of unmatched. I don't anticipate this to be confusing to users, but wanted to point it out.

If a user has a file named '.foo.conf' it will not be matched, nor will '.foo.bar' be shown as being skipped.